java/network/cnp-ingress-jetty-tcp-protocol.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: cnp-ingress-jetty-tcp-protocol
```

Comment:
@tamilmaran-7 Please add the main workload name to the policy name. Since we don't have tags in cilium its good if we have workload name in policy name. After ingress you can add the workload name.

Reply:
done @salman-accuknox